spec/sec-regexp-regular-expression-objects-patch.html

```diff
@@ -104,50 +104,111 @@ <h1>Runtime Semantics: BackreferenceMatcher ( _n_, _direction_ )</h1>
   </emu-clause>
 </emu-clause>
 
+<emu-clause id="sec-regexp-abstract-operations">
+  <h1>RegExp Abstract Operations</h1>
+
+  <emu-clause id="sec-matchoptions-records">
+    <h1><ins>MatchOptions Records</ins></h1>
+    <ins>
+    <p>A MatchOptions is a Record value used to encapsulate options for a regular expression matching operation.</p>
+    <p>MatchOptions Records have the fields listed in <emu-xref href="#table-matchoptions-record"></emu-xref>.</p>
+    </ins>
+    <emu-table id="table-matchoptions-record" caption="MatchOptions Record Fields">
+      <table>
+        <tbody>
+          <tr>
+            <th>Field Name</th>
+            <th>Value</th>
+            <th>Meaning</th>
+          </tr>
+          <tr>
+            <td>[[Captures]]</td>
+            <td>`"strings"` or `"indices"`</td>
+            <td>Whether the operation should capture substrings or the start/end indices of the capture.</td>
+          </tr>
+        </tbody>
+      </table>
+    </emu-table>
+  </emu-clause>
+
+  <emu-clause id="sec-frommatchoptions" aoid="FromMatchOptions">
+    <h1><ins>FromMatchOptions ( _Options_ )</ins></h1>
+    <p>
+      When the abstract operation FromMatchOptions is called with argument _Options_, the following steps are taken:
+    </p>
+    <emu-alg>
+    1. If _Options_ is *undefined*, return *undefined*.
+    1. Assert: _Options_ is a MatchOptions Record.
+    1. Let _obj_ be ObjectCreate(%ObjectPrototype%).
+    1. Assert: _obj_ is an extensible ordinary object with no own properties.
+    1. If _Options_ has a [[Capture]] field, then
```

Comment:
when would a MatchOptions Record not have a [[Capture]] field? The assertion on step 2 should guarantee this condition is always true.